.github/workflows/bitbot.yaml

```diff
@@ -0,0 +1,187 @@
+name: Bitbot
+on:
+  issue_comment:
+    types:
+      - created
+      - edited
+
+  pull_request_review_comment:
+    types:
+      - created
+      - edited
+
+  pull_request_review:
+    types:
+      - submitted
+      - edited
+
+  pull_request_target:
+    types:
+      - opened
+
+  issues:
+    types:
+      - opened
+
+  schedule:
+    - cron: '0 * * * *'
+
+  workflow_dispatch:
+
+env:
+  # This commands is for anyone who can use it
+  COMMON_PLUGINS: |-
+    assign
+    unassign
+    kind
+    remove-kind
+    cc
+    uncc
+
+  # This commands is for author of issue or PR
+  AUTHOR_PLUGINS: |-
+    retest
+    close
+    reopen
+
+  # This commands is for organization member or repository member
+  MEMBERS_PLUGINS: |-
+    good-first-issue
+    help-wanted
+    close
+    reopen
+
+  # This commands is for in the REVIEWERS environment variable
+  REVIEWERS_PLUGINS: |-
+    area
+    remove-area
+    lgtm
+    hold
+    retitle
+
+  # This commands is for in the APPROVERS environment variable
+  APPROVERS_PLUGINS: |-
+    approve
+
+  # This commands is for in the MAINTAINERS environment variable
+  MAINTAINERS_PLUGINS: |-
+    milestone
+    remove-milestone
+    priority
+    remove-priority
+
+  # This commands is for in the ADMINS environment variable
+  ADMINS_PLUGINS: |-
+
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GH_REPOSITORY: ${{ github.repository }}
+  GH_DEBUG: true
+
+# https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+permissions:
```

Comment:
can you elaborate on why this is needed ?
would prefer if we started off with minimal permissions

Reply:
Yes, I will try to minimize the permissions the action will be granted.